### PR TITLE
fix: update methods for basket-bar visibility to use route. ensure it…

### PR DESCRIPTION
**Pull request by mcstover**
…'s hidden on checkout

### CODE REVIEW

File: src/components/WwwFrame/TheBasketBar.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
 	<!-- eslint-disable max-len -->
 	<div
-		v-if="count > 0"
-		class="tw-flex md:tw-hidden tw-fixed tw-z-sticky"
+		v-if="count > 0 && !hideBasketBar"
+		class="basket-bar tw-flex md:tw-hidden tw-fixed tw-z-sticky"
```
Comment (mcstover) [inline src/components/WwwFrame/TheBasketBar.vue:5]: Prob could have fixed things by just adding this class but worked up the no css approach instead. Then realized this class is used for locating this component from the monthly good selector that can show on homepage so had to add it anyway...

File: src/components/WwwFrame/TheBasketBar.vue [hunk 2]
```diff
@@ -69,6 +69,15 @@ export default {
 	computed: {
 		basketLink() {
 			return this.corporate ? this.addHashToRoute('show-basket') : '/basket';
+		},
+		hideBasketBar() {
+			// hide this banner on managed lending landing + checkout pages
+			const routeExclusions = ['/join-team', '/checkout'];
```
Comment (ryan-ludwig) [inline src/components/WwwFrame/TheBasketBar.vue:75]: We have some route exclusions on `TheBasketBar.vue`'s parent, `WwwPage.vue` that determines if basket bar should be on the page. I think these route exclusions would make more sense to go there.

Comment (ryan-ludwig) [reply src/components/WwwFrame/TheBasketBar.vue:75]: Or we consolidate those route rules into this component instead.

**Review by emuvente — APPROVED** [review]
👍

**Review by ryan-ludwig — APPROVED** [review]

**Pull request merged**